weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/SchemaMappingStep.tsx

```diff
@@ -0,0 +1,352 @@
+import {Box, Paper, Stack, Tooltip, Typography} from '@mui/material';
```

Comment:
![Screenshot 2025-02-04 at 18 13 07](https://github.com/user-attachments/assets/a34829c3-58bc-4584-b47f-491aeeba8ce2)

I think the call id pills are not super useful and clutter the screen

Reply:
Agreed, ripped these out